### ISSUE TITLE
W1b: size-argument order differs — `bbox_to_mask(boxes, width, height)` vs `Boxes.to_mask(height, width)`

### ISSUE THREAD

**Issue by ducha-aiki** [kornia/kornia#4014]
The free function and the method that produce the same mask take the image size in opposite orders. Everywhere else in kornia sizes are `(height, width)` — `dsize`, `create_meshgrid(height, width)`, `Boxes3D.to_mask(depth, height, width)`, and `bbox_to_mask3d(boxes, size=(depth, height, width))` in the *same* module — so `bbox_to_mask` is the odd one out. This is the live form of the "x,y vs H,W ordering" question from #1142 (closed); the plan's repair-window inventory had no open issue for it.

## Repro

```python
import torch, inspect
from kornia.geometry.bbox import bbox_to_mask, bbox_to_mask3d
from kornia.geometry.boxes import Boxes, Boxes3D

v = torch.tensor([[[1., 1.], [4., 1.], [4., 3.], [1., 3.]]])
print("bbox_to_mask(v, 5, 3).shape:", tuple(bbox_to_mask(v, 5, 3).shape), " Boxes(v).to_mask(3, 5).shape:", tuple(Boxes(v).to_mask(3, 5).shape))
print("bbox_to_mask(v, 3, 5).shape:", tuple(bbox_to_mask(v, 3, 5).shape), " Boxes(v).to_mask(5, 3).shape:", tuple(Boxes(v).to_mask(5, 3).shape))
for f in (bbox_to_mask, Boxes.to_mask, bbox_to_mask3d, Boxes3D.to_mask):
    print(f.__qualname__, inspect.signature(f))
```

## Observed

```
bbox_to_mask(v, 5, 3).shape: (1, 3, 5)  Boxes(v).to_mask(3, 5).shape: (1, 3, 5)
bbox_to_mask(v, 3, 5).shape: (1, 5, 3)  Boxes(v).to_mask(5, 3).shape: (1, 5, 3)
bbox_to_mask (boxes, width: int, height: int)
Boxes.to_mask (self, height: int, width: int)
bbox_to_mask3d (boxes, size: tuple[int, int, int])          # (depth, height, width)
Boxes3D.to_mask (self, depth: int, height: int, width: int)
```

Both produce `(B, H, W)`; the *argument order* is the trap. A caller who moves from the method to the function with a non-square image gets a transposed mask, no error.

## Expected

`(height, width)` everywhere, matching `dsize` and the 3D forms. Swapping two positional parameters is a silent semantic change for every existing `bbox_to_mask` caller (their masks transpose), so it needs a deprecation path (keyword-only transition or a new name) inside the window; the batch-4 convention blocks state the order explicitly on both symbols meanwhile.

## Classification

**Semantic contract (argument-order convention).** Idiom: none of the §2b idioms — an ordering convention, recorded as the resolution of the #1142 ordering question.

## Disposition

**MERGE IN WINDOW — behavior change.** Canonical: `(height, width)` (the `dsize` order used by every other size argument in kornia, including `bbox_to_mask3d` and both 3D methods). Shape: align contract, both names kept. Divergence today (executed): `bbox_to_mask(v, 5, 3)` and `Boxes(v).to_mask(3, 5)` are the same call. This is the behavior-change discussion CONTRIBUTING asks for, and the decision is: **in the coordinated Repair & Deprecation window, not before. A PR that aliases or consolidates these ahead of it will be closed and pointed here.**

## Environment

kornia 0.9.0rc1 @ `0685b1f5` (main), torch 2.9.1, python 3.11, cpu. Found during the bbox/boxes conventions audit (batch 4, tracking issue #4020); the batch documents the behavior below as it is and pins it, and does not change it.

Posted on behalf of @ducha-aiki by Claude (Fable 5).
